pysb/bng.py

```diff
@@ -137,8 +137,18 @@ def _format_action_args(cls, **kwargs):
             Arguments and values to supply to BNG
         """
         if kwargs:
-            action_args = ','.join('%s=>%s' % (k, BngConsole._bng_param(v))
-                                   for k, v in kwargs.items())
+            processed_pars = []
+            for k, v in kwargs.items():
+                bng_param = BngConsole._bng_param(v)
+                if isinstance(bng_param, dict):
+                    par_string = f'{k}' + '=>{' + ','.join(f"{k}=>{v}" for k, v in bng_param.items()) + '}'
+                    processed_pars.append(par_string)
```

Comment:
This line could be factored out - it applies to both `if/else` branches

Reply:
yep